packages/zosfiles/__tests__/cli/unmount/fs/fs.definition.unit.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition, ICommandOptionDefinition } from "@zowe/imperative";
+
+describe("zos-files unmount fs command definition", () => {
+    it("should not have changed", () => {
+        const definition: ICommandDefinition = require("../../../../src/cli/unmount/fs/fs.definition").FsDefinition;
+
+        expect(definition).toBeDefined();
+
+        // Should not contain children since this is a command
+        expect(definition.children).toBeUndefined();
+
+        // Should require a zosmf profile
+        expect(definition.profile.optional).toEqual(["zosmf"]);
+
+        // Should only contain one positional
+        expect(definition.positionals.length).toEqual(1);
+
+        // The positional should be required
+        expect(definition.positionals[0].required).toBeTruthy();
```

Comment:
This may be a nitpick, but the value should be exactly `true`. I'm sure we have this style expect everything in the code though.